flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/xa/JdbcXaSinkMigrationTest.java

```diff
@@ -77,7 +79,8 @@ public JdbcXaSinkMigrationTest(FlinkVersion readVersion) {
 
     private final FlinkVersion readVersion;
 
-    @Test
+    @TestTemplate
+    @Disabled
```

Comment:
I have to disable this as getReadVersions return a empty list..
This method appear to be used only manually as nothing is tested in normal execution.

I try to return some FlinkVersion but resources are missing :S
I get this error:
`java.io.FileNotFoundException: src/test/resources/jdbc-exactly-once-sink-migration-1.16-snapshot (No such file or directory)`

Reply:
I would like to remove the disable (as it will unable to test manually without touch it) but I dont see how..
Or if is not used at all, remove the class